cli.js

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env node
+'use strict';
+
+var path = require('path');
+var childProcess = require('child_process');
+var inquirer = require('inquirer');
+var assign = require('lodash.assign');
+var utils = require('./cli-utils');
+var codemods = require('./codemods.json');
+
+function runScripts(scripts, files) {
+	var spawnOptions = {
+		env: assign({}, process.env, {PATH: path.resolve('node_modules/.bin') + ':' + process.env.PATH})
+	};
+
+	var result;
+	scripts.forEach(function (script) {
+		result = childProcess.spawnSync('jscodeshift', ['-t', script].concat(files.split(' ')), spawnOptions);
```

Comment:
That is fragile as they could change the internal name at any time. Better to use https://github.com/sindresorhus/npm-run-path#usage and you can keep spawning it by name.